ingestion/src/metadata/ingestion/source/database/redshift/metadata.py

```diff
@@ -448,10 +448,14 @@ def get_partition_details(self) -> None:
         """
         Populate partition details
         """
-        self.partition_details.clear()
-        results = self.engine.execute(REDSHIFT_PARTITION_DETAILS).fetchall()
-        for row in results:
-            self.partition_details[f"{row.schema}.{row.table}"] = row.diststyle
+        try:
+            self.partition_details.clear()
+            results = self.engine.execute(REDSHIFT_PARTITION_DETAILS).fetchall()
+            for row in results:
+                self.partition_details[f"{row.schema}.{row.table}"] = row.diststyle
+        except Exception as exe:
+            logger.debug(traceback.format_exc())
+            logger.debug(f"Failed to fetch partition details due: {exe}")
```

Comment:
debug or warning?

Reply:
I was not sure if we want to log this information as this is an optional step which fetches the partition details, which for now is only used by the profiler IG and not displayed on the UI... also the prime reason of the failure would be that the used does not have access to svv_info_table and that will be highlighted in the test connection ... do you think we should make it warning?

Reply:
Fair enough! We can leave it as it is.